systemtests/src/main/java/io/enmasse/systemtest/platform/apps/SystemtestsKubernetesApps.java

```diff
@@ -481,22 +507,41 @@ private static Deployment getMessagingAppDeploymentResource() {
                 .endSelector()
                 .withReplicas(1)
                 .withNewTemplate()
-                .withNewMetadata()
-                .addToLabels("app", MESSAGING_CLIENTS)
-                .endMetadata()
-                .withNewSpec()
-                .addNewContainer()
-                .withName(MESSAGING_CLIENTS)
-                .withImage("quay.io/enmasse/systemtests-clients:latest")
-                .withCommand("sleep")
-                .withArgs("infinity")
-                .endContainer()
-                .endSpec()
-                .endTemplate()
+                    .withNewMetadata()
+                    .addToLabels("app", MESSAGING_CLIENTS)
+                    .endMetadata()
+                    .withNewSpec()
+                    .addNewContainer()
+                    .withName(MESSAGING_CLIENTS)
+                    .withImage("quay.io/enmasse/systemtests-clients:latest")
+                    .withCommand("sleep")
+                    .withArgs("infinity")
+                    .endContainer()
+                    .endSpec()
+                    .endTemplate()
                 .endSpec()
                 .build();
     }
 
+    private static Deployment getMessagingAppDeploymentResource(String namespace) {
+       Deployment deployment = new DoneableDeployment(getMessagingAppDeploymentResource())
```

Comment:
You can also use DeploymentBuilder().
...
build().

Which I've seen used in most parts of the code, but I'm fine leaving it as is also, it works :)

Reply:
I will change it to builder, nice catch :)